alma/alma-monthlypayments-magento2
# Bump version 4.1.0

### Reason for change

Merge Dev to main : 
feature : remove flag inpage
fix checkout js bug, selectPaymentMethod()



### Code changes



### How to test
Check in page in admin with an account without feature flag
Check payment method selection : select in first a not alma payment method and after select A alma payment method

_As a reviewer, you are encouraged to test the PR locally._


No UI change

### Checklist for authors and reviewers




### Non applicable
